docs/mobile-apps/automated-testing/appium/appium-versions.md

```diff
@@ -133,8 +133,7 @@ capabilities.AddAdditionalCapability("sauce:options", sauceOptions);
 
 Our Appium 2 release strategy is as follows:
 
-- On the 2nd of each month, we release a 'bundled' version of Appium 2 that includes all the latest drivers available up to the 1st of that month. This 'bundled' version allows you to maintain a stable testing environment, saving you the effort required to individually update drivers. The version is named with a timestamp in the `appiumVersion` capability, such as `appium2-YYYYMMDD`, for easy tracking. These versions have an 11-month lifespan post-release, giving you plenty of time to plan your upgrade strategy.
-
+- We offer a `stable` image, which is our LTS image. We change this image as conservatively as possible. The regular update schedule is 2 years. We will inform you when this image will be updated and offer a migration guide for all breaking changes from one `stable` image to a new `stable` image. We can unfortunately not guarantee that the `stable` image will never change, because the OS versions of our devices naturally evolve, which introduces breaking changes to drivers. If OS changes break some of our core device functionality we will update the drivers in our `stable` image.
```

Comment:
"We can unfortunately not guarantee that the `stable` image will never change, because the OS versions of our devices naturally evolve, which introduces breaking changes to drivers."

Lets change this to 

"The stable image is subject to change, to ensure the latest OS versions and device models are continuously supported with the latest Appium drivers."